tests/form-filling.js

```diff
@@ -0,0 +1,104 @@
+const helper = require('./helper');
```

Comment:
This file is doing a lot of the filling work and is the one  I'm most concerned about.

Reply:
I'm going to submit my other comments so you can have a look at that, but I'll come back to this and make sure I give it proper attention.